app/models/post.rb

```diff
@@ -0,0 +1,11 @@
+class Post < ApplicationRecord
+  belongs_to :user
+
```

Comment:
- [ ] Also, you have not associated the `post` model with `likes` and `comments`. Please add association here between `post and comment` and `post and likes`


```ruby
class Post < ApplicationRecord
 ..
 has_many :comments
 has_many :likes
end
```